src/test/skript/tests/misc/comments.sk

```diff
@@ -0,0 +1,30 @@
+test "comments":
+	parse:
+		set {_a} to {_b} # test
+		set {_a} to {_b} ## test
+		set {_a} to {_b} ### test
+	assert last parse logs is not set with "skript should be able to handle inline comments with multiple hashtags but did not"
+
+
+	parse:
+		set {_a} to "a"
+		set {_a} to "#a"
+		set {_a} to "##a"
+		set {_a} to "###a"
```

Comment:
Can we test that these actually are `###a`, etc. in case the hashtags are being squeezed together still (by magic somehow idk, just think it'd be good to test this as much as possible)? -- Actually we might have to test this with char-at or something, since anything that was done wrong in the original would also be wrong in the assertion.

Also do you think we could try some longer and more hashtaggy patterns to see if we can stress the parser into breaking? 🤣

Reply:
Well the above issue is covered with comments.sk, and yeah, i can test for compression.
I highly doubt there would be any issues as to the parser breaking with more hashtags
it's strictly O(n) and only does simple char checks for each character once, so i think we'd need like massive lines to get any reaction out of the parser.

Reply:
Right, I was just thinking that maybe it might struggle to count properly in a whole mess of percents and hashtags and quotes and an actual comment thrown in there for good measure, I know I would.
Feel free to dismiss this if you think it's ok and I'll approve.

Reply:
I'll throw in the compression tests and one nightmare string for good measure